### DISCUSSION TITLE
General Question: Whether to renew/maintain dotspatial.org domain name?

### DISCUSSION THREAD

**Discussion by danames** [DotSpatial/DotSpatial#1416] [General]
This is a general question to the current/active DotSpatial developers and users. Is there value in keeping/maintaining the DotSpatial.org web domain? This domain simply redirects to the GitHub pages for the project. I've had the domain for nearly 10 years and it is up for renewal and I can continue to maintain it but I suspect most users and developers just go directly to the GitHub pages, not through the dotspatial.org domain. Anyone with an opinion on this, please comment on this issue with your opinion. Thanks, Dan

**Comment by sindizzy** [comment]
The only reason I would hold on to it would be for "brand recognition". Some other person could buy it and use it for reasons that wouldn't bode well on the project.

**Comment by jany-tenaj** [comment]
I see it like sindizzy.

**Comment by danames** [comment]
OK I'll close this issue. I ended up renewing dotspatial.org and I have it set up as a redirect to the github page.
